### ISSUE TITLE
what is the pretrained model accuracy?

### ISSUE THREAD

**Issue by ycyeu** [nyoki-mtl/keras-facenet#25]
I need a baseline thanks.
